### PR TITLE
feat: added version number and ascii art addition

### LINKED ISSUE #106: Feature: Include version number in the console.
### Type of feature

🍕 Feature

### Current behavior

This app relies on semantic versioning and it would be ideal to let the user know what version they are using. 

### Suggested solution

In the flagship app we highlight the version in the console. 
Screen Shot 2022-01-23 at 8 00 09 PM

1. Include [fetch version lib](https://github.com/open-sauced/open-sauced/blob/main/src/lib/appVersion.js) similar to open sauced.
2. [Add ascii art](https://github.com/open-sauced/open-sauced/blob/eb1bb9030cbc0bba6d883d5ed933e99f5488ea14/src/index.jsx#L43-L49) to the index page. 

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by erinmikailstaples**
## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description

Please do not leave this blank 
This PR updates version as well as adds the ascii art to the index page.
-

## Related Tickets & Documents
Fixes #106
https://github.com/open-sauced/hot/issues/106

## Mobile & Desktop Screenshots/Recordings

Workflow

Set up:

Forked repo, [cloned it here.](https://github.com/erinmikailstaples/hot-add-version-number-console)
Task 1: Include [fetch version lib](https://github.com/open-sauced/open-sauced/blob/main/src/lib/appVersion.js) similar to open sauced.

Added new file under lib > appVersion.js and copied over the version page
Image

Task 2: [Add ascii art](https://github.com/open-sauced/open-sauced/blob/eb1bb9030cbc0bba6d883d5ed933e99f5488ea14/src/index.jsx#L43-L49) to the index page.

Added ascii art to the index page here:
Image

Add ascii art https://github.com/open-sauced/hot/issues/106#issuecomment-1217018904
Opened src > app.tsx + added the ascii art starting on line 13 (view below)
Image

Syncing to the [repo here](https://github.com/erinmikailstaples/hot-add-version-number-console) now.

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [x] 🙋 no, because I need help
_i think i did this right... maybe?_

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Review by erinmikailstaples — COMMENTED** [review]
Removed it in index.js + renamed PR

File: src/App.tsx [hunk 1]
```diff
@@ -10,6 +10,19 @@ import GradBackground from "./components/GradBackground";
 import Hero from "./components/Hero";
 import apiFetcher from "./hooks/useSWR";
 
+import { version } from "../../package.json";
```
Comment (0-vortex) [inline src/App.tsx:13]:
```
import { version } from "../package.json";
```

File: src/App.tsx [hunk 2]
```diff
@@ -10,6 +10,19 @@ import GradBackground from "./components/GradBackground";
...
+
+    console.log(`%c
+ ██████╗ ██████╗ ███████╗███╗   ██╗    ███████╗ █████╗ ██╗   ██╗ ██████╗███████╗██████╗
+██╔═══██╗██╔══██╗██╔════╝████╗  ██║    ██╔════╝██╔══██╗██║   ██║██╔════╝██╔════╝██╔══██╗
+██║   ██║██████╔╝█████╗  ██╔██╗ ██║    ███████╗███████║██║   ██║██║     █████╗  ██║  ██║
+██║   ██║██╔═══╝ ██╔══╝  ██║╚██╗██║    ╚════██║██╔══██║██║   ██║██║     ██╔══╝  ██║  ██║
+╚██████╔╝██║     ███████╗██║ ╚████║    ███████║██║  ██║╚██████╔╝╚██████╗███████╗██████╔╝
+ ╚═════╝ ╚═╝     ╚══════╝╚═╝  ╚═══╝    ╚══════╝╚═╝  ╚═╝ ╚═════╝  ╚═════╝╚══════╝╚═════╝%c v${version()}`,
```
Comment (0-vortex) [inline src/App.tsx:21]: `version` instead on `version()` - it's a string value, calling it as a string will remove the error

File: src/lib/appVersion.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+import pkg from "../../package.json";
```
Comment (0-vortex) [inline src/lib/appVersion.js:1]: this file is not used right now - consider getting the version from this file import in `App.tsx` or removing this file entirely

**Review by 0-vortex — CHANGES_REQUESTED** [review]
Minor changes should remove the lint issues, then running an `npm run format` should auto-apply our code formatter and make everything green! 🍕 

Doing these changes should unblock the deployment preview so you can see the version in the console - I tested locally with the proposed changes and it LGTM! 👍

File: index.html [hunk 4]
```diff
@@ -34,6 +34,19 @@
...
+    console.log(`%c
+ ██████╗ ██████╗ ███████╗███╗   ██╗    ███████╗ █████╗ ██╗   ██╗ ██████╗███████╗██████╗
+██╔═══██╗██╔══██╗██╔════╝████╗  ██║    ██╔════╝██╔══██╗██║   ██║██╔════╝██╔════╝██╔══██╗
+██║   ██║██████╔╝█████╗  ██╔██╗ ██║    ███████╗███████║██║   ██║██║     █████╗  ██║  ██║
+██║   ██║██╔═══╝ ██╔══╝  ██║╚██╗██║    ╚════██║██╔══██║██║   ██║██║     ██╔══╝  ██║  ██║
+╚██████╔╝██║     ███████╗██║ ╚████║    ███████║██║  ██║╚██████╔╝╚██████╗███████╗██████╔╝
+ ╚═════╝ ╚═╝     ╚══════╝╚═╝  ╚═══╝    ╚══════╝╚═╝  ╚═╝ ╚═════╝  ╚═════╝╚══════╝╚═════╝%c v${version()}`,
+    "color:#f6d82b",
+    "color:green;font-weight:bold");
+
```
Comment (bdougie) [inline index.html:49]:
```suggestion
```

We can remove this

Comment (bdougie) [inline index.html:49]: This is not needed let's removes it🙏🏾

**Review by bdougie — CHANGES_REQUESTED** [review]
Can you run `npm run format` locally and then push that up? It will get rid of some errors.

**Comment by bdougie** [comment]
@erinmikailstaples heads up, I fixed your PR on stream and open a new one here. #332 

For context I did the following
- Ran npm run format
- Replaced the package.json version with appVersion.ts

**Pull request closed without merging**